### PR TITLE
Add LDAP case and add delay to project config to avoid case fail

### PR DESCRIPTION
**Pull request by sigsbee**

### CODE REVIEW

File: tests/robot-cases/Group11-Nightly/LDAP.robot [hunk 1]
```diff
@@ -37,6 +37,42 @@ Test Case - Ldap Sign in and out
...
+    Page Should Not Contain  mike01
+    Back To Projects
+    Create An New Project  project${d}
+    Go Into Project  project${d}
+    Switch To Member
+    Add Guest Member To Project  mike02
+    Page Should Contain  mike02
+    Close Browser
+
+Test Case - Link For DB Should Not Exist
```
Comment (wy65701436) [inline tests/robot-cases/Group11-Nightly/LDAP.robot]: Test Case - Home Page Differences With DB Mode

File: tests/robot-cases/Group11-Nightly/LDAP.robot [hunk 2]
```diff
@@ -37,6 +37,42 @@ Test Case - Ldap Sign in and out
...
+    Init Chrome Driver
+    sign in harbor  ${HARBOR_URL}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}
```
Comment (wy65701436) [inline tests/robot-cases/Group11-Nightly/LDAP.robot]: Sign In Harbor

File: tests/robot-cases/Group11-Nightly/LDAP.robot [hunk 3]
```diff
@@ -37,6 +37,42 @@ Test Case - Ldap Sign in and out
...
+    Logout Harbor
+    Sleep  2
+    Page Should Not Contain  Sign up 
+    Page Should Not Contain  Forgot password
+    Close Browser
+
+Test Case - Ldap Add User Button Disabled
```
Comment (wy65701436) [inline tests/robot-cases/Group11-Nightly/LDAP.robot]: Test Case - New User Is Unusable

File: tests/robot-cases/Group11-Nightly/LDAP.robot [hunk 4]
```diff
@@ -37,6 +37,42 @@ Test Case - Ldap Sign in and out
...
+    Init Chrome Driver
+    Sign In Harbor  ${HARBOR_URL}  ${HARBOR_ADMIN}  ${HARBOR_PASSWORD}
+    Switch To User Tag
+    Add User Button Should Be Disabled
+    Close Browser
+
+Test Case Ldap User Can Not Change Password
```
Comment (wy65701436) [inline tests/robot-cases/Group11-Nightly/LDAP.robot]: Test Case - Change Password Is Invisible

File: tests/resources/Harbor-Pages/LDAP-Mode.robot [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+*** Settings ***
+Documentation  This resource provides any keywords related to the Harbor private registry appliance
+Resource  ../../resources/Util.robot
+
+*** Variables ***
+${HARBOR_VERSION}  v1.1.1
+
+*** Keywords ***
+
+Ldap User Should Not See Change Password
```
Comment (wy65701436) [inline tests/resources/Harbor-Pages/LDAP-Mode.robot]: please move this method to UserProfile.robot, and rename it as "Change Password Is Invisible"

File: tests/resources/Harbor-Pages/LDAP-Mode.robot [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+    Click Element  //clr-header//clr-dropdown[2]//button
+    Sleep  1
+    Page Should Not Contain  Password
+
+Add User Button Should Be Disabled
```
Comment (wy65701436) [inline tests/resources/Harbor-Pages/LDAP-Mode.robot]: please move this method to Administration-Users.robot, and rename it as "New User Is Unusable"

File: tests/robot-cases/Group11-Nightly/LDAP.robot [hunk 7]
```diff
@@ -37,6 +37,42 @@ Test Case - Ldap Sign in and out
     Sign In Harbor  ${HARBOR_URL}  mike  zhu88jie
     Close Browser
 
+Test Case - Add A Never Login Member
```
Comment (wy65701436) [inline tests/robot-cases/Group11-Nightly/LDAP.robot]: Test Case - Sys Admin On-board New Member

and add one more case to on-board member via project admin.

File: tests/robot-cases/Group11-Nightly/LDAP.robot (hunk 3 above)
Comment (sigsbee) [reply tests/robot-cases/Group11-Nightly/LDAP.robot]: IT'S A **BUTTON**, **NOT** A USER

File: tests/resources/Harbor-Pages/LDAP-Mode.robot (hunk 6 above)
Comment (sigsbee) [reply tests/resources/Harbor-Pages/LDAP-Mode.robot]: AGAIN, IT"S A **BUTTON**, **NOT** A USER

**Pull request merged**